AppConfig.js

```diff
@@ -87,10 +87,10 @@ var config = {
         urlBTC: 'https://cex.io/api/last_price/DASH/BTC',
         urlEur: 'https://cex.io/api/last_price/DASH/EUR'
       },
-      yahoofinance: {
-        name: 'Yahoo',
-        orgUrl: 'https://www.yahoo.com/',
-        url: 'https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20yahoo.finance.xchange%20where%20pair%20in%20(%22KRWUSD%22)&format=json&env=store://datatables.org/alltableswithkeys&callback='
+      currencyconverterapi: {
```

Comment:
Why don't you use camelCase? It's more readable.

Reply:
fixed